src/utils/mod.rs

```diff
@@ -274,6 +308,117 @@ impl TpmsRsaParmsBuilder {
 /// Supported sizes for RSA key modulus
 pub const RSA_KEY_SIZES: [u16; 4] = [1024, 2048, 3072, 4096];
 
+/// Builder for `TPMS_ECC_PARMS` values.
+// Most of the field types are from bindgen which does not implement Debug on them.
+#[allow(missing_debug_implementations)]
```

Comment:
This is no longer the case.

Reply:
Nice! Does that mean we could remove this `allow`?

Reply:
Yes, and put `Debug` in derive